src/PluginClient.php

```diff
@@ -62,9 +65,20 @@ public function __construct($client, array $plugins = [], array $options = [])
      */
     public function sendRequest(RequestInterface $request)
     {
-        $promise = $this->sendAsyncRequest($request);
+        if (!($this->client instanceof HttpClient)) {
+            return $this->sendAsyncRequest($request)->wait();
+        }
+
+        $client = $this->client;
+        $pluginChain = $this->createPluginChain($this->plugins, function (RequestInterface $request) use ($client) {
```

Comment:
The difference is if you have an async client and a sync client, it will use the correct underlying method for sendRequest or sendAsyncRequest, before it was always using the sendAsyncRequest one.